policies/api-compat.md

```diff
@@ -0,0 +1,15 @@
+Policy on API compatibility in minor releases
+=============================================
+
+**No changes in existing public API functions and data are allowed.**
+
+Only API additions are allowed in minor releases.
+
+Although there might be some kinds of changes that are ABI compatible such
```

Comment:
Does this mean constification is allowed?  Allowing it will break code that takes the address of a function.

if so, I'd suggest splitting this into two sentences.  The first being "Constification of parameters is allowed" or similar.

Reply:
No, nothing is allowed except additions of new things.

Reply:
It reads as if some changes are permitted.

How about:
> No changes to any existing API are permitted.  This includes, but is not limited to:
> * constification of arguments;
> * changing void returns to int returns;
> * changing a macro to a function and
> * corrections of spelling.

Reply:
OK, done.